test.md

```diff
@@ -270,6 +270,38 @@ print it and run it through the `markdownify` Liquid filter. Here's a demo:
 </div>
 </div>
 
+#### Synchronizing multiple tab groups
```

Comment:
I think it's dangerous to do string sub-parsing like the other option you outlined. Since Bootstrap doesn't have this built in, we would have had to make something up anyway. :)